src/main/webapp/app/course/manage/detail/course-detail-line-chart.component.ts

```diff
@@ -75,20 +76,28 @@ export class CourseDetailLineChartComponent extends ActiveStudentsChart implemen
 
     constructor(private service: CourseManagementService, private translateService: TranslateService) {
         super();
+        this.translateService.onLangChange.subscribe(() => {
+            this.updateYAxisLabel();
+        });
     }
 
     ngOnChanges() {
-        this.loading = true;
         this.amountOfStudents = this.translateService.instant('artemisApp.courseStatistics.amountOfStudents');
-        // Only use the pre-loaded stats once
-        if (this.initialStatsReceived || !this.initialStats) {
+        this.updateYAxisLabel();
+        this.determineDisplayedPeriod(this.course, this.displayedNumberOfWeeks);
+        /*if (this.showLifetimeOverview) {
             return;
+        }*/
+        if (this.currentOffsetToEndDate === 0 || !this.course.startDate) {
+            this.displayDefaultChartScope();
+            /*
+            if the course has a start date and already ended
+            (i.e. the time difference between today and the course end date is at least one week), we show the lifetime overview by default.
+             */
+        } else {
+            this.showLifetimeOverview = true;
+            this.displayLifetimeOverview();
         }
```

Comment:
```suggestion
        /*
        if the course has a start date and already ended
        (i.e. the time difference between today and the course end date is at least one week), we show the lifetime overview by default.
         */
        if (this.course.startDate && this.currentOffsetToEndDate !== 0) {
            this.showLifetimeOverview = true;
            this.displayLifetimeOverview();
        } else {
            this.displayDefaultChartScope();
        }
```
Switching the then/else-blocks make the code directly follow the logic described in the comment.